webrtc.html

```diff
@@ -1807,8 +1807,8 @@
                         </ol>
                       </li>
                       <li>
-                        <p>If <code><var>candidate</var>.ufrag</code> is
-                        non-null, and is not equal to any ufrag present in the
+                        <p>If <code><var>candidate</var>.ufrag</code> is not
+                        missing, and is not equal to any ufrag present in the
```

Comment:
But, actually... `ufrag` _is_ nullable in the interface version of `RTCIceCandidate`. So if it's missing from the dictionary it will be `undefined`, and if it's missing from the interface it will be `null`, correct?

In that case, is "set" suitable, in order to avoid a double-negative, or is "not missing" most accurate?